scss/lab.scss

```diff
@@ -21,6 +21,18 @@ iframe {
   transition: 1s width linear;
 }
 
+iframe.small {
```

Comment:
Hm yes but:
It should work by thursday anything else can be refactored.
(I think I will get to it wednesday)

Id say leave it for now - refactor later